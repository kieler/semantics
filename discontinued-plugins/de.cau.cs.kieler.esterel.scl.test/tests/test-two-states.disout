module TWO_STATES:
input On;
input Off;
output IsOn;
output IsOff;

loop
  abort
    loop
      emit IsOff;
      pause
    end loop
  when On;
  abort
    loop
      emit IsOn;
      pause
    end loop
  when Off
end loop

end module
