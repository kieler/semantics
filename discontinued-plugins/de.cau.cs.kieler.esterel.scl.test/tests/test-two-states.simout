   0 IsOn=0 IsOff=1 
   1 IsOn=1 IsOff=0 
   2 IsOn=1 IsOff=0 
   3 IsOn=1 IsOff=0 
   4 IsOn=1 IsOff=0 
   5 IsOn=1 IsOff=0 
   6 IsOn=1 IsOff=0 
   7 IsOn=1 IsOff=0 
   8 IsOn=0 IsOff=1 
   9 IsOn=0 IsOff=1 
  10 IsOn=1 IsOff=0 
  11 IsOn=1 IsOff=0 
  12 IsOn=0 IsOff=1 
  13 IsOn=0 IsOff=1 
  14 IsOn=0 IsOff=1 
  15 IsOn=0 IsOff=1 
  16 IsOn=1 IsOff=0 
  17 IsOn=0 IsOff=1 
  18 IsOn=1 IsOff=0 
  19 IsOn=0 IsOff=1 
  20 IsOn=0 IsOff=1 
  21 IsOn=1 IsOff=0 
  22 IsOn=0 IsOff=1 
  23 IsOn=1 IsOff=0 
  24 IsOn=1 IsOff=0 
  25 IsOn=0 IsOff=1 
  26 IsOn=0 IsOff=1 
  27 IsOn=0 IsOff=1 
  28 IsOn=1 IsOff=0 
  29 IsOn=1 IsOff=0 
  30 IsOn=1 IsOff=0 
  31 IsOn=1 IsOff=0 
  32 IsOn=0 IsOff=1 
  33 IsOn=1 IsOff=0 
  34 IsOn=1 IsOff=0 
  35 IsOn=0 IsOff=1 
  36 IsOn=0 IsOff=1 
  37 IsOn=1 IsOff=0 
  38 IsOn=1 IsOff=0 
  39 IsOn=0 IsOff=1 
  40 IsOn=0 IsOff=1 
  41 IsOn=1 IsOff=0 
  42 IsOn=0 IsOff=1 
  43 IsOn=1 IsOff=0 
  44 IsOn=0 IsOff=1 
  45 IsOn=0 IsOff=1 
  46 IsOn=1 IsOff=0 
  47 IsOn=0 IsOff=1 
  48 IsOn=0 IsOff=1 
  49 IsOn=0 IsOff=1 
  50 IsOn=0 IsOff=1 
  51 IsOn=0 IsOff=1 
  52 IsOn=1 IsOff=0 
  53 IsOn=1 IsOff=0 
  54 IsOn=0 IsOff=1 
  55 IsOn=1 IsOff=0 
  56 IsOn=0 IsOff=1 
  57 IsOn=0 IsOff=1 
  58 IsOn=1 IsOff=0 
  59 IsOn=0 IsOff=1 
  60 IsOn=0 IsOff=1 
  61 IsOn=1 IsOff=0 
  62 IsOn=1 IsOff=0 
  63 IsOn=1 IsOff=0 
  64 IsOn=1 IsOff=0 
  65 IsOn=1 IsOff=0 
  66 IsOn=0 IsOff=1 
  67 IsOn=1 IsOff=0 
  68 IsOn=1 IsOff=0 
  69 IsOn=1 IsOff=0 
  70 IsOn=0 IsOff=1 
  71 IsOn=0 IsOff=1 
  72 IsOn=1 IsOff=0 
  73 IsOn=0 IsOff=1 
  74 IsOn=1 IsOff=0 
  75 IsOn=1 IsOff=0 
  76 IsOn=0 IsOff=1 
  77 IsOn=0 IsOff=1 
  78 IsOn=0 IsOff=1 
  79 IsOn=1 IsOff=0 
  80 IsOn=0 IsOff=1 
  81 IsOn=1 IsOff=0 
  82 IsOn=0 IsOff=1 
  83 IsOn=0 IsOff=1 
  84 IsOn=0 IsOff=1 
  85 IsOn=1 IsOff=0 
  86 IsOn=1 IsOff=0 
  87 IsOn=0 IsOff=1 
  88 IsOn=0 IsOff=1 
  89 IsOn=1 IsOff=0 
  90 IsOn=1 IsOff=0 
  91 IsOn=1 IsOff=0 
  92 IsOn=0 IsOff=1 
  93 IsOn=1 IsOff=0 
  94 IsOn=1 IsOff=0 
  95 IsOn=1 IsOff=0 
  96 IsOn=1 IsOff=0 
  97 IsOn=1 IsOff=0 
  98 IsOn=0 IsOff=1 
  99 IsOn=1 IsOff=0 
