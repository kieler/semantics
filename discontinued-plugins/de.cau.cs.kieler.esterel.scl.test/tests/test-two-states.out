module TWO_STATES:
input On;
input Off;
output IsOn;
output IsOff;

loop
  abort
    sustain IsOff
  when On;
  abort
    sustain IsOn
  when Off
end loop

end module
